### PR TITLE
Server setup

### PR DESCRIPTION
**Pull request by carlitoswillis**
- added a simple server
- add index.jsx, set paths in config and package.json

### CODE REVIEW

File: client/components/info.jsx [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+    return (
+      <div>
+        <h3> Product </h3>
+      </div>
+    );
+  }
+
+}
+
+export default Info;
```
Comment (AmiraVomir) [inline client/components/info.jsx:23]: add extra line at end

Comment (carlitoswillis) [reply client/components/info.jsx:23]: there is one :+) line 24 is an empty new line

**Comment by AmiraVomir** [comment]
@carlitoswillis package-lock.json should not be .gitignored. It is intended to be commit according to npm documentations

**Comment by carlitoswillis** [comment]
> @carlitoswillis package-lock.json should not be .gitignored. It is intended to be commit according to npm documentations

appreciate the info! makes sense- also gonna unhide webpack config.

**Pull request merged**